classes/hooks/synchronization/Initial.php

```diff
@@ -83,18 +83,13 @@ public function synchronize()
             array('Datatype' => 'str', 'Name' => $segmentationObject->ll(49), 'NameSpace' => 'static')
         ));
 
-        $file = new SplFileObject('contacts.csv', 'w');
         $headers = array("email","firstname","lastname");
-        $file->fputcsv($headers);
+        $contstToAddCsvString = '';
```

Comment:
Don`t you know that if some variable is not declared and then you try to add a value to it will cause a PHP warning ?
From PHP.NET :  E_NOTICE level error is issued in case of working with uninitialized variables.